.github/CODEOWNERS

```
@@ -1579,6 +1579,7 @@ packages/kbn-monaco/src/esql @elastic/kibana-esql
 /.eslintignore @elastic/kibana-operations
 
 # QA - Appex QA
+/x-pack/plugins/maps/ui_tests @elastic/appex-qa # temporarily
```

The maps plugin has been moved since the creation of this PR, which has been picked up inthe code changes already. However, the code owners entry needs to be updated as well:
```suggestion
/x-pack/platform/plugins/shared/maps/ui_tests @elastic/appex-qa # temporarily
```